tools/uswds-boilerplate-on-codepen.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: USWDS Boilerplate on CodePen
+nav: Tools
+permalink: /tools/uswds-boilerplate-on-codepen/
+sidenav: tools-side-nav
+---
+# {{ page.title }}
+
+CodePen is an web-based editor for experimenting with and prototyping HTML, CSS, and JavaScript. It's the easiest way to start getting familiar with USWDS code snippets and design tokens because you don't have to worry about dev environments and hosting. All you do is put some code in the releavnt boxes and it just works.
+
+## Getting started
+
+1. You can edit CodePens without an account, but since you'll probably want to save your work, we recommend you [sign up for a free account](https://codepen.io/accounts/signup/user/free).
+1. Open our [USWDS Boilerplate pen](https://codepen.io/pglevy/pen/abBgJbe).
+1. To save a copy, press the `fork` button on the right side of the toolbar at the bottom of the page.
+1. The name will default to the same as the original, but you can rename it by pressing the pencil icon next to the name.
+
+![Default view of the USWDS Boilerplate pen on CodePen]({{ site.baseurl }}/assets/img/boilerplate-editors.png){: .border-base-light .border-1px}
+
+## Understanding the settings
+
+- This pen has all the settings needed to use the default version of USWDS.
+- If you want, you can view these by pressing the `Settings` button.
+- The HTML settings link to a hosted version of the CSS.
+- The JS settings link to the a hosted version of the JS file that is needed for some components to work.
+- There are no CSS settings by default.
+
+![CodePen settings panel]({{ site.baseurl }}/assets/img/pen-settings.png){: .border-base-light .border-1px}
+
+## Understanding the default code
+
+- This boilerplate makese use of CodePen's include feature, which allows you render code from one pen inside another.
```

Comment:
Add 'to' between you and render. So it reads, 'allows you to render' 
Do you think we made need more clarity in this section? Now that I know more about the process it makes sense to me. But I wonder for someone who is new to Codepen if this would be confusing, even though they don't necessarily need to do anything here. I wonder if people would need more clarity to what the include feature is?

Reply:
Also, teeny typo: ...makes* use of CodePen's include...

Reply:
And to Demi's point, maybe just adding a resource link to the [documentation](https://blog.codepen.io/documentation/adding-external-resources/#html-can-be-an-external-resource-too-6) page for this?

Something like, "if you want to know more, go here" blurb.

Reply:
Fixed typos and added inline link to docs